certified-connectors/document-ai-konfuzio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,623 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Document AI Konfuzio",
+    "termsOfService": "https://konfuzio.com/impressum/",
+    "contact": {
+      "name": "Helm & Nagel GmbH",
+      "email": "info@konfuzio.com",
+      "url": "https://help.konfuzio.com"
+    },
+    "license": {
+      "name": "Helm & Nagel GmbH"
+    },
+    "version": "v2",
+    "description": "Build custom Document AI to ocr, categorize and extract any document."
+  },
+  "host": "app.konfuzio.com",
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://konfuzio.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://konfuzio.com/de/impressum/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Data;Content and Files"
+    }
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v2/docs/": {
+      "post": {
+        "operationId": "v2_docs_create",
+        "summary": "Upload a new document",
+        "description": "Upload a new document",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "x-ms-summary": "File",
+            "description": "File to send to Host.",
+            "in": "formData",
+            "name": "data_file",
+            "type": "file",
+            "required": true
+          },
+          {
+            "x-ms-summary": "Project ID",
+            "description": "The ID of your project.",
+            "in": "formData",
+            "name": "project",
+            "type": "integer",
+            "required": true
+          },
+          {
+            "x-ms-summary": "Synchron response",
+            "description": "Default is False",
+            "in": "formData",
+            "name": "sync",
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "",
```

Comment:
@atraining Please provide valid description value for all the response codes in the swagger.

Reply:
I added the description